### PR TITLE
V3: Rename the zFTP extension

### PR DESCRIPTION
**Pull request by zFernand0**
## Proposed changes

Rename the zFTP extension to `IBM z/OS FTP for Zowe Explorer`

## Release Notes

Milestone: v3 pre-release

Changelog: Renamed extension 😋 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking change which adds or improves functionality)
- [ ] Breaking change (a change that would cause existing functionality to not work as expected)
- [ ] Documentation (Markdown, README updates)
- [x] Other (please specify above in "Proposed changes" section)

## Checklist

**General**

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] All PR dependencies have been merged and published (if applicable)
- [ ] A GIF or screenshot is included in the PR for visual changes
- [x] The pre-publish command has been executed:
  - **v2 and below:** `yarn workspace vscode-extension-for-zowe vscode:prepublish`
  - **v3:** `pnpm --filter vscode-extension-for-zowe vscode:prepublish`

**Code coverage**

- [ ] There is coverage for the code that I have added
- [ ] I have added new test cases and they are passing
- [ ] I have manually tested the changes

**Deployment**

- [ ] I have added developer documentation (if applicable)
- [ ] Documentation should be added to Zowe Docs
  - If you're an outside contributor, please post in the [#zowe-doc Slack channel](https://openmainframeproject.slack.com/archives/CC961JYMQ) to coordinate documentation.
  - Otherwise, please check with the rest of the squad about any needed documentation before merging.
- [x] These changes may need ported to the appropriate branches (list here):
  - [ ] `next`

### CODE REVIEW

File: packages/zowe-explorer-ftp-extension/CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ All notable changes to the "zowe-explorer-ftp-extension" extension will be docum
 - Changed the hashing algorithm for e-tag generation from `sha1` to `sha256` to avoid collisions. [#2890](https://github.com/zowe/zowe-explorer-vscode/pull/2890)
 - Updated the FTP plugin dependency to `3.0.0-next.202403191358` for technical currency [#2783](https://github.com/zowe/vscode-extension-for-zowe/pull/2783).
 - Update Zowe SDKs to `8.0.0-next.202405241828` for technical currency. [#2918](https://github.com/zowe/zowe-explorer-vscode/issues/2918)
+- Renamed extension to `IBM z/OS FTP Plug-in for Zowe`. [#2990](https://github.com/zowe/zowe-explorer-vscode/issues/2990)
```
Comment (t1m0thyj) [inline packages/zowe-explorer-ftp-extension/CHANGELOG.md:14]: Should this say `IBM z/OS FTP for Zowe Explorer` to match the new display name? 😋

File: packages/zowe-explorer-ftp-extension/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ All notable changes to the "zowe-explorer-ftp-extension" extension will be docum
...
+
+## `2.17.0`
```
Comment (t1m0thyj) [inline packages/zowe-explorer-ftp-extension/CHANGELOG.md:16]: Is the added header intentional?

**Review by t1m0thyj — COMMENTED** [review]
Should we also update the header in `README.md`, and the display name in `.github/release.config.js`?

(Same question as for the V2 PR 😋)

**Review by traeok — APPROVED** [review]

**Review by awharn — DISMISSED** [review]

**Review by awharn — APPROVED** [review]

**Review by traeok — APPROVED** [review]

**Comment by zFernand0** [comment]
ZE doesn't need a changelog update for simple change in a unite test file or for zFTP changes 😋 
Merging! 🥳

**Pull request merged**